R/plot_metrics.R

```diff
@@ -75,12 +75,12 @@ plot_metrics <- function(test_path, metric, num_commits = 5, save_data = FALSE,
   stopifnot(is.logical(save_plots))
   stopifnot(length(save_plots) == 1)
   floor(num_commits)
-  
+
   if (metric == "time") {
     if (interactive) {
       temp_out <- capture.output(.plot_interactive_time(test_path, num_commits, save_data, save_plots))
     } else {
-      temp_out <- capture.output(.plot_time(test_path, num_commits, save_data, save_plots))
+       temp_out <- capture.output(.plot_time(test_path, num_commits, save_data, save_plots))      
```

Comment:
please make indentation consistent (here there are three spaces for else but only two for if)

Reply:
Sure. I will resolve the indentation and push on the **test-pkg** branch.